static/bidder-info/colossus.yaml

```diff
@@ -0,0 +1,11 @@
+maintainer:
+  email: "aigolkin1991@gmail.com"
```

Comment:
This looks like a personal email address. Do you have a mailing list or work email from colossus to use instead?